app/segments/(team)/chooseTeam.js

```diff
@@ -141,6 +180,35 @@ function ChooseTeam() {
             justifyContent: "center",
           }}
         >
+          <Button
```

Comment:
Maybe we can not display the "Join team" button until the user is verified?

Reply:
Sure, I don't mind hiding that button